CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.3.1]
+
+### Changed
+
+  - Fixes issue where Picture-in-Picture does not work with FireworkVideoSDK 1.3.0 and later.
```

Comment:
maybe 
"Picture-in-Picture in story block does not work with FireworkVideoSDK 1.3.0 and later."
should be better.